opencti-platform/opencti-front/src/private/components/settings/Workflow.js

```diff
@@ -80,6 +86,8 @@ class Sessions extends Component {
             return <div />;
           }}
         />
+        <WorkflowsStatusesMenu />
+        {view === 'lines' ? this.renderLines(paginationOptions) : ''}
```

Comment:
We try as much as possible to drop this 'view === 'lines'' stuff as it is useless